DEVELOPMENT.md

```diff
@@ -5,65 +5,76 @@ Errors are done with `thiserror`. We adopt a minimalistic approach to errors.
 * Errors that can be handled and reasoned upon should be able to be matched and extract good knowledge from
-* To users, error should _not be cryptic_, and should indicate how to fix issues as much as possible, or point to the issue precisely
-
+* To users, error should _not be cryptic_, and should indicate how to fix issues as much as possible, or point to the
+  issue precisely
 
 ### Auto conversions
 
 When possible use `from` conversions.
 
 ```rust
-    #[error(transparent)]
-    JSON(#[from] serde_json::Error),
+    # [error(transparent)]
+JSON( # [from] serde_json::Error),
 ```
 
-When complicated, implement a `From` trait yourself. This is done to _centralize_ errors into one place and not litter needless `map_err` code which holds error conversion logic (an exception is Context, see below).
-
+When complicated, implement a `From` trait yourself. This is done to _centralize_ errors into one place and not litter
+needless `map_err` code which holds error conversion logic (an exception is Context, see below).
 
 ### Context
 
-When you know a user might need context, resort to manually shaping the error with extra information. First, define the error:
+When you know a user might need context, resort to manually shaping the error with extra information. First, define the
+error:
 
 ```rust
-    #[error("cannot parse `{1}`: {0}")]
-    YAMLFile(#[source] serde_yaml::Error, String),
+    # [error("cannot parse `{1}`: {0}")]
+YAMLFile( # [source] serde_yaml::Error, String),
 ```
 
 Then, shape it:
 
 ```rust
-  serde_yaml::from_str(&rendered)
-      .map_err(|err| Error::YAMLFile(err, selected_path.to_string_lossy().to_string()))
+  serde_yaml::from_str( & rendered)
+.map_err( | err| Error::YAMLFile(err, selected_path.to_string_lossy().to_string()))
 ```
 
-In this example, the information about where `rendered` came from was long lost at the `serde_yaml::from_str` callsite. Which is why errors were cryptic indicating bad YAML format, but not where it comes from (which file).
+In this example, the information about where `rendered` came from was long lost at the `serde_yaml::from_str` callsite.
+Which is why errors were cryptic indicating bad YAML format, but not where it comes from (which file).
 
-In this case, we duplicate the YAML error type, leave one of those for auto conversions with `from`, where we don't have a file, and create a new specialized error type with the file information: `YAMLFile`.
+In this case, we duplicate the YAML error type, leave one of those for auto conversions with `from`, where we don't have
+a file, and create a new specialized error type with the file information: `YAMLFile`.
 
 ## Publishing a new version
 
 **Test your changes**
 
 * [ ] run `cargo test` on the root to test Loco itself
+* [ ] ```$ docker run -d -p 5432:5432 -e POSTGRES_USER=loco -e POSTGRES_DB=loco_app -e POSTGRES_PASSWORD="loco" postgres:15.3-alpine```
```

Comment:
what are you trying to say here?

Reply:
Added more text, to to notify need the `docker` up, it's in the main `README.md` but not here, so adding for completeness.

Reply:
While it's beneficial to enhance our text content,I believe we should aim to avoid command duplicates for improved maintainability. Consider the PR #216 as an example, we have new feature that  dynamically generate the database name based on the application name and environment in the starter. 

To optimize this, how about incorporating a concise instruction like:

"Ensure you have the necessary local resources, such as db/redis, by executing the command `cargo loco doctor`. In case you don't have them, refer to the relevant documentation section for guidance."

WDYT?

Reply:
Updated.